From b589f702f7570ec4f649014665ba16ac1c9b5bec Mon Sep 17 00:00:00 2001
From: "ji, zhenlong z" <zhenlong.z.ji@intel.com>
Date: Fri, 17 Sep 2021 10:25:04 +0800
Subject: [PATCH] Disable restrictions for debugfs

Google doesn't encourage the vendor to use debugfs any
longer for security issues. However, there are still some
domains(graphics) in Celadon need to use debugfs to
implement some functions. Will push the relevant domain
owners to abandon the use of debugfs in future.

Tracked-On: OAM-99326
Signed-off-by: ji, zhenlong z <zhenlong.z.ji@intel.com>
---
 groups/device-specific/caas/caas.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/groups/device-specific/caas/caas.mk b/groups/device-specific/caas/caas.mk
index a3d92fb..7ef34e7 100644
--- a/groups/device-specific/caas/caas.mk
+++ b/groups/device-specific/caas/caas.mk
@@ -59,6 +59,7 @@ PRODUCT_AAPT_CONFIG := normal large mdpi
 PRODUCT_AAPT_PREF_CONFIG := mdpi
 
 PRODUCT_RESTRICT_VENDOR_FILES := false
+PRODUCT_SET_DEBUGFS_RESTRICTIONS := false
 {{^ota-update}}
 PRODUCT_SHIPPING_API_LEVEL := 30
 PRODUCT_OTA_ENFORCE_VINTF_KERNEL_REQUIREMENTS := true
-- 
2.25.1

